modules/virt-troubleshooting-incorrect-policy-config.adoc

```diff
@@ -66,12 +72,15 @@ However the Policy status alone does not indicate if it failed on all nodes or a
 
 . List the Enactments to see if the Policy was successful on any of the nodes. If the Policy failed for only a subset it suggests the problem is with specific node configuration; if the Policy failed on all nodes it suggest the problem is with the Policy.
 +
+[source,terminal]
 ----
 $ oc get nnce
 ----
 +
 The output shows that the Policy failed on all nodes:
 +
+.Example output
+[source,terminal]
 ----
 NAME                                   STATUS
 utput
```

Comment:
utput?

this clearly wasn't part of this PR, but it seems like a typo

Reply:
Niiiice catch.
That shouldn't be there at all.
Gone